app.py

```diff
@@ -59,7 +65,6 @@ def fetch_html_content(url):
     headers = {
         'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3',
         'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8',
-        'Accept-Encoding': 'gzip, deflate, br',
```

Comment:
Nit: neden?

Reply:
bu olunca compressed data , donuyormus. bazen oluyor (yani sana denk gelmemıs olabilir). bunu silmeden önce kod çalışmıyordu. response.text sacma bır seydi
https://stackoverflow.com/questions/55441361/python-requests-response-not-in-correct-encoding